Concentration/Concentration.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Concentration.swift
+//  Concentration
+//
+//  Created by Oren Dinur on 08/02/2022.
+//
+
+import Foundation
+
+class Concentration{
+    
+    var cards = Array <Card>()
+    //Same as [Card]()
+    
+    var flipCount = 0
+    
+    var score = 0
+    
+    var seenCards = [Int:Int]()
+
+    var indexOfOneAndOnlyFaceUpCard: Int?
+     
+    //The beginning of our game
+    var previousClick = Date()
```

Comment:
Do we want to save time at this point? If yes, maybe better to initialize it in the `init` method.